src/turbulence_closures/verstappen_anisotropic_minimum_dissipation.jl

```diff
@@ -1,30 +1,39 @@
-struct VerstappenAnisotropicMinimumDissipation{T} <: AbstractAnisotropicMinimumDissipation{T}
-     C :: T
-    Cb :: T
-     ν :: T
-     κ :: T
-end
-
 """
-    VerstappenAnisotropicMinimumDissipation(T=Float64; C=1/12, ν=1.05e-6, κ=1.46e-7)
```

Comment:
Note that this function signature is still correct, because its provided via the function `typed_keyword_constructor`.